util/grpc/src/retry_config.rs

```diff
@@ -0,0 +1,45 @@
+use retry::delay;
+use serde::Serialize;
+use std::time::Duration;
+use structopt::StructOpt;
+
+/// An object which represents a retry policy for retriable errors for a grpc
+/// connection
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize, StructOpt)]
```

Comment:
Basic question: What would we gain from supporting Serialize but not Deserialize?